data/json/flags.json

```diff
@@ -64,6 +64,15 @@
     "description": "Blinds the wearer while worn, and provides nominal protection vs flashbang flashes.",
     "info": "This gear <bad>prevents</bad> you from <info>seeing</info> anything."
   },
+  {
+    "id": "EFFECT_NIGHT_VISION",
+    "type": "json_flag",
+    "context": [ "EFFECT" ],
+    "conflicts": [ "BLIND" ],
+    "description": "Allows normal sight without external light source.",
+    "//": "Supported by color in propeties",
+    "properties": { "color": "green" }
```

Comment:
What is that for? "properties" are only loaded for item types. On effects, it's ignored.